Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,87 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
+import CalcDisplay from "./components/DisplayComponents/CalculatorDisplay";
+import ActionButton from "./components/ButtonComponents/ActionButton";
+import NumberButton from "./components/ButtonComponents/NumberButton";
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  state = {
+    total: 2
+  };
 
+  Clear = () => {
+    console.log(this.state);
+    if (this.state.total > 0) {
+      this.setState({
+        total: 0
+      });
+    }
+  };
+
+  render() {
+    return (
+      <div className="calc-container">
+        <CalcDisplay text={this.state.total} />
+        <ActionButton
+          buttonStyle="action-btn"
+          text="clear"
+          click={this.Clear}
+        />
+        <NumberButton buttonStyle="operator" text="&divide;" />
```

Comment:
Good layout, but could be made drier via judicious use of map and a number/operator array.